lib/get/get-full-source-space.js

```diff
@@ -166,5 +166,5 @@ function extractItems (response) {
 }
 
 function filterDrafts (items, includeDrafts) {
-  return includeDrafts ? items : items.filter((item) => item.sys.publishedVersion)
+  return includeDrafts ? items : items.filter((item) => !!item.sys.publishedVersion)
```

Comment:
Do you remember what was the issue that required this explicit casting to boolean?